src/qt/walletview.cpp

```diff
@@ -122,11 +122,12 @@ void WalletView::setWalletModel(WalletModel *_walletModel)
     overviewPage->setWalletModel(_walletModel);
     receiveCoinsPage->setModel(_walletModel);
     sendCoinsPage->setModel(_walletModel);
-    usedReceivingAddressesPage->setModel(_walletModel->getAddressTableModel());
-    usedSendingAddressesPage->setModel(_walletModel->getAddressTableModel());
 
     if (_walletModel)
```

Comment:
After a quick research it looks like `_walletModel` is never passed as NULL. So it would be good to change the argument from a pointer to a reference and remove this check.